Sources/CrossPlatformExtensions/typealiases.swift

```diff
@@ -0,0 +1,21 @@
+#if canImport(Cocoa)
+import Cocoa
+#elseif canImport(UIKit)
+import UIKit
+#endif
+
+#if canImport(SwiftUI)
+import SwiftUI
+#endif
+
+#if canImport(Cocoa)
+public typealias Window = NSWindow
```

Comment:
I was worried about using simple/elegant names like this as they would be in the way of the user's class being named so locally. Prefixing with the module name is not that great either.

One possibility would be to call this a `UIWindow` on the mac too?